CIP-XXXX/README.md

```diff
@@ -0,0 +1,96 @@
+---
+CIP: XX
+Title: Modular Exponentiation Built-in for Plutus Core
+Status: Proposed
+Category: Plutus
+Authors:
+    - Kenneth MacKenzie <kenneth.mackenzie@iohk.io>
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+    - Iñigo Querejeta-Azurmendi <inigo.querejeta@iohk.io>
+    - Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Implementors:
+    - IOG
```

Comment:
Ideally, implementors aren't companies but _people_, so we can refer to them when needed. Would it be okay to put @michaelpj here?

Reply:
Put us both I guess